### ISSUE TITLE
No models better than random

### ISSUE THREAD

**Issue by StevenLOL** [automl/auto-sklearn#512]
Why I have the following warnings:

```
[WARNING] [2018-07-20 11:11:50,755:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] No models better than random - using Dummy Classifier!
[WARNING] [2018-07-20 11:11:50,785:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] No models better than random - using Dummy Classifier!
[WARNING] [2018-07-20 11:11:52,790:EnsembleBuilder(1):bc7ea84b06c06db9da18252766d56c64] No models better than random - using Dummy Classifier!
```
Code:
```
auto=AutoSklearnClassifier(include_estimators=['lda','k_nearest_neighbors','passive_aggressive','gradient_boosting','extra_trees','liblinear_svc'], 
                              resampling_strategy='cv',
                            resampling_strategy_arguments={'folds': 5},
)
```

**Comment by mfeurer** [comment]
This happens in the beginning when the ensemble script does not find any model which performs better than random. It should go away after some time.

**Comment by btwewek** [comment]
Hi, I have the same problem when running the example:
```
X, y = sklearn.datasets.load_breast_cancer(return_X_y=True)
X_train, X_test, y_train, y_test = \
    sklearn.model_selection.train_test_split(X, y, random_state=1)

automl = autosklearn.classification.AutoSklearnClassifier(
    time_left_for_this_task=1200,  
    ensemble_size=1,
    include_estimators=['xgradient_boosting'],
)
```
If I setting the ensemble_size to 1 and only one estimators included, did it run the hyper-parameter optimization(SMAC) to that single estimator many times than return the best?
It would always output following when use xgboost :
`[WARNING] [2019-06-10 18:32:29,928:EnsembleBuilder(1):breast_cancer] No models better than random - using Dummy Classifier!
`
also the final model performance was bad.
But when I changed xgradint_boosting to extra_tress, it works.

**Comment by FlandrinorZXY** [comment]
> Hi, I have the same problem when running the example:
> 
> ```
> X, y = sklearn.datasets.load_breast_cancer(return_X_y=True)
> X_train, X_test, y_train, y_test = \
>     sklearn.model_selection.train_test_split(X, y, random_state=1)
> 
> automl = autosklearn.classification.AutoSklearnClassifier(
>     time_left_for_this_task=1200,  
>     ensemble_size=1,
>     include_estimators=['xgradient_boosting'],
> )
> ```
> 
> If I setting the ensemble_size to 1 and only one estimators included, did it run the hyper-parameter optimization(SMAC) to that single estimator many times than return the best?
> It would always output following when use xgboost :
> `[WARNING] [2019-06-10 18:32:29,928:EnsembleBuilder(1):breast_cancer] No models better than random - using Dummy Classifier! `
> also the final model performance was bad.
> But when I changed xgradint_boosting to extra_tress, it works.

I met the same problem with xgb and also I tried to add a new lgbmregressor according to the example,finally I got a the same result again……
if there can be any example about adding lgbRgressor?

**Comment by mfeurer** [comment]
There is most likely an issue with the latest release of XGBoost (#698) which is the reason why using only XGBoost results in a bad score. We're working on replacing XGBoost by scikit-learn's new HistGradientBoosting, which is based upon LGBT.

**Comment by datavisoryangzhou** [comment]
I met the same issue by including only "random_forest" and "xgradient_boosting":
`automl = autosklearn.classification.AutoSklearnClassifier(
    time_left_for_this_task = 3600,
    per_run_time_limit = 360,
    include_estimators=["random_forest", "xgradient_boosting", ], 
    include_preprocessors=["no_preprocessing", ], 
    seed = 42,
    ml_memory_limit = 30720,
    resampling_strategy = 'holdout',
    n_jobs=8
)`

Any suggestion on how to make auto-sklearn work if I want to use tree based model?

**Comment by mfeurer** [comment]
Could you please check the output of `print(automl.sprint_statistics())`? Maybe the time limits are set too low.

**Comment by mfeurer** [comment]
We removed XGBoost in the latest 0.6.0 release. Therefore, I'm closing this. Please reopen if this issue still persists with the latest version.
